lib/commands/util/upgrade-project/hosted-skill-helper.js

```diff
@@ -130,9 +131,13 @@ function postUpgradeGitSetup(profile, doDebug, gitClient, callback) {
     _setPrePushHookTemplate(profile, doDebug, (hooksErr) => callback(hooksErr || null));
 }
 
-function _setGitCredentialHelper(profile, gitClient) {
-    const credentialHelperPath = `ask util git-credentials-helper --profile ${profile}`;
-    gitClient.updateCredentialHelper(credentialHelperPath);
+function _setGitCredentialHelper(profile, gitClient, repositoryUrl, skillId) {
+    const credentialScriptPath = path.join(os.homedir(),
+        CONSTANTS.FILE_PATH.ASK.HIDDEN_FOLDER,
+        CONSTANTS.FILE_PATH.ASK.SCRIPTS_FOLDER.NAME,
+        CONSTANTS.FILE_PATH.ASK.SCRIPTS_FOLDER.GIT_CREDENTIAL_HELPER);
+    const credentialScriptExecution = `'${credentialScriptPath}' ${profile} ${skillId}`;
+    gitClient.configureCredentialHelper(credentialScriptExecution, repositoryUrl);
```

Comment:
To configure git-credential-helper script while upgrading a v1 skill project to v2.